CONTRIBUTING.md

```diff
@@ -43,3 +46,39 @@ make test-integration
 * Commits should be as small as possible, while ensuring that each commit is correct independently.
 
 * Add tests relevant to the fixed bug or new feature.
+
+
+## Troubleshooting build
+
+### docker approach
+you can build in a docker container by going to root dir of gopass project and:
+```
+docker run --rm -v "$PWD":/go/src/github.com/justwatchcom/gopass -w /go/src/github.com/justwatchcom/gopass golang:stretch go build -v
+```
+
+replace `go build -v` in above command with `make test` or anything else you'd like to run inside docker.
```

Comment:
Capitalization. Also I'd replace `anything else` with something along the line of `any other command`.